roles/openshift_logging_eventrouter/tasks/main.yaml

```diff
@@ -0,0 +1,88 @@
+# allow passing in a tempdir
+- name: Create temp directory for doing work in
+  command: mktemp -d /tmp/openshift-logging-ansible-XXXXXX
+  register: mktemp
+  changed_when: False
+
+- set_fact:
+    tempdir: "{{ mktemp.stdout }}"
```

Comment:
Consider change the name to be something more specific to the role.  I believe if this role is run with other roles it will clobber any other use of the name 'tempdir'

Reply:
I am not sure I understand, other roles seem to have implemented the creation of tempdir exactly the same way. Could you please elaborate what am I missing? Thank you

- https://github.com/openshift/openshift-ansible/blob/d57f27bf71f68ba91996bc4a0b1e79b3be956b7e/roles/openshift_logging_mux/tasks/main.yaml#L19
- https://github.com/openshift/openshift-ansible/blob/d57f27bf71f68ba91996bc4a0b1e79b3be956b7e/roles/openshift_logging_curator/tasks/main.yaml#L11
- https://github.com/openshift/openshift-ansible/blob/d57f27bf71f68ba91996bc4a0b1e79b3be956b7e/roles/openshift_logging_elasticsearch/tasks/main.yaml#L27
- https://github.com/openshift/openshift-ansible/blob/d57f27bf71f68ba91996bc4a0b1e79b3be956b7e/roles/openshift_logging_fluentd/tasks/main.yaml#L43
- https://github.com/openshift/openshift-ansible/blob/d57f27bf71f68ba91996bc4a0b1e79b3be956b7e/roles/openshift_logging_kibana/tasks/main.yaml#L13

Reply:
I defer to your finding as it appears we use the same pattern without incident